### PR TITLE
Now filtering todo list only for owned items

### PR DESCRIPTION
**Pull request by aremo-ms**
## Purpose

Fileting ToDo list only to signed in user ownership

### CODE REVIEW

File: TodoListClient/Controllers/TodoListController.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ public ActionResult Index()
             //reset session on every entry to TODO's list
             TodoSessionState(SessionAction.Set);
 
-            return View(_commonDBContext.Todo.ToList());
+            return View(_commonDBContext.Todo.Where(l => l.Owner.Equals(HttpContext.User.Identity.Name)).ToList());
```
Comment (jmprieur) [inline TodoListClient/Controllers/TodoListController.cs]: Filtering is authorization.
It should not be done with the Name claim. but with the oid+tid. Please use [HttpContext.User.GetMsalAccountId()](https://github.com/AzureAD/microsoft-identity-web/blob/d26e79ddf9fdd2e1fa740711ec652cb2a221fc1f/src/Microsoft.Identity.Web.TokenCache/ClaimsPrincipalExtensions.cs#L80)

**Review by jmprieur — CHANGES_REQUESTED** [review]
The code is not secure

File: TodoListClient/Controllers/TodoListController.cs (hunk 1 above)
Comment (aremo-ms) [reply TodoListClient/Controllers/TodoListController.cs]: @jmprieur I'm learning new things today. Thanks for input

**Comment by aremo-ms** [comment]
@jmprieur please take a peek
CC: @kalyankrishna1

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @aremo-ms

**Pull request merged**